index.html

```diff
@@ -69,107 +77,124 @@ <h1 class="intro-text">I’m Mwape Glad to see you!</h1>
+              <li class="dot"></li>
+              <li class="light-text common">Back End Dev</li>
+              <li class="dot"></li>
+              <li class="light-text common">2021</li>
+            </ul>
+
+            <p class="decription-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-list">
+              <li><span>html</span></li>
+              <li class="hiden-on-mobile"><span>Ruby on Rails</span></li>
+              <li><span>css</span></li>
+              <li><span>JavaScript</span></li>
+            </ul>
+            <button class="see-project" type="button">See Project</button>
+          </div>
         </div>
       </section>
       <section class="projects">
         <div class="project-card">
           <div class="screenshot-project">
-            <img src="assets/project1.png" alt="" />
+            <img src="assets/card-pro.png" alt="" />
           </div>
-          <h2 class="works-text">Tonic</h2>
-          <ul class="about-me-btns">
-            <li class="block-text">CANOPY</li>
-            <li class="dot"></li>
-            <li class="light-text commons">Back End Dev</li>
-            <li class="dot"></li>
-            <li class="light-text commons">2021</li>
-          </ul>
 
-          <p class="decription-text">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="tech-list">
-            <li><span>html</span></li>
-            <li><span>css</span></li>
-            <li><span>JavaScript</span></li>
-          </ul>
-          <button class="see-project" type="button">See Project</button>
+          <div class="project-text">
+            <h2 class="works-text">Tonic</h2>
+            <ul class="about-me-btns">
+              <li class="block-text">CANOPY</li>
+              <li class="dot"></li>
+              <li class="light-text common">Back End Dev</li>
+              <li class="dot"></li>
+              <li class="light-text common">2021</li>
+            </ul>
+
+            <p class="decription-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-list">
+              <li><span>html</span></li>
+              <li class="hiden-on-mobile"><span>Ruby on Rails</span></li>
+              <li><span>css</span></li>
+              <li><span>JavaScript</span></li>
+            </ul>
+            <button class="see-project" type="button">See Project</button>
+          </div>
         </div>
       </section>
+
       <section class="projects">
         <div class="project-card">
-          <div class="screenshot-project">
-            <img src="assets/project2.png" alt="" />
+          <div class="screenshot-project order-2@md">
+            <img src="assets/cardi-pic.png" alt="" />
```

Comment:
I will suggest you always add text to alt in <img src="assets/cardi-pic.png" alt="" /> element, this text will show if the image did not load.